src/services/UserService.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'src/config/axios/axios';
+
+export const getUser = async (id: string) => {
+  const response = await axios.get('/users/' + id);
```

Comment:
No need for variable this should be `return response = await axios.get('/users/' + id);`

Reply:
Is this okay `return await axios.get(`/users/${id}`);`?